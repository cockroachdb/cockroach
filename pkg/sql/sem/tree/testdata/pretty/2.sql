SELECT count(*) count, winner, counter * 60 * 5 as counter FROM (SELECT winner, round(length / 60 / 5) as counter FROM players WHERE build = $1 AND (hero = $2 OR region = $3)) GROUP BY winner, counter
