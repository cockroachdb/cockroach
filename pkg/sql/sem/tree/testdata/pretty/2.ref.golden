// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
SELECT
	count(
		*
	)
		AS count,
	winner,
	counter
	* 60
	* 5
		AS counter
FROM
	(
		SELECT
			winner,
			round(
				length
				/ 60
				/ 5
			)
				AS counter
		FROM
			players
		WHERE
			build
			= $1
			AND (
					hero
					= $2
					OR region
						= $3
				)
	)
GROUP BY
	winner,
	counter

12:
------------
SELECT
	count(*)
		AS count,
	winner,
	counter
	* 60
	* 5
		AS counter
FROM
	(
		SELECT
			winner,
			round(
				length
				/ 60
				/ 5
			)
				AS counter
		FROM
			players
		WHERE
			build
			= $1
			AND (
					hero
					= $2
					OR region
						= $3
				)
	)
GROUP BY
	winner,
	counter

19:
-------------------
SELECT
	count(*)
		AS count,
	winner,
	counter
	* 60
	* 5
		AS counter
FROM
	(
		SELECT
			winner,
			round(
				length
				/ 60
				/ 5
			)
				AS counter
		FROM
			players
		WHERE
			build
			= $1
			AND (
					hero
					= $2
					OR region
						= $3
				)
	)
GROUP BY
	winner, counter

20:
--------------------
SELECT
	count(*)
		AS count,
	winner,
	counter * 60 * 5
		AS counter
FROM
	(
		SELECT
			winner,
			round(
				length
				/ 60
				/ 5
			)
				AS counter
		FROM
			players
		WHERE
			build
			= $1
			AND (
					hero
					= $2
					OR region
						= $3
				)
	)
GROUP BY
	winner, counter

22:
----------------------
SELECT
	count(*) AS count,
	winner,
	counter * 60 * 5
		AS counter
FROM
	(
		SELECT
			winner,
			round(
				length
				/ 60
				/ 5
			)
				AS counter
		FROM
			players
		WHERE
			build = $1
			AND (
					hero
					= $2
					OR region
						= $3
				)
	)
GROUP BY
	winner, counter

29:
-----------------------------
SELECT
	count(*) AS count,
	winner,
	counter * 60 * 5
		AS counter
FROM
	(
		SELECT
			winner,
			round(
				length
				/ 60
				/ 5
			)
				AS counter
		FROM
			players
		WHERE
			build = $1
			AND (
					hero = $2
					OR region
						= $3
				)
	)
GROUP BY
	winner, counter

31:
-------------------------------
SELECT
	count(*) AS count,
	winner,
	counter * 60 * 5 AS counter
FROM
	(
		SELECT
			winner,
			round(
				length / 60 / 5
			)
				AS counter
		FROM
			players
		WHERE
			build = $1
			AND (
					hero = $2
					OR region
						= $3
				)
	)
GROUP BY
	winner, counter

34:
----------------------------------
SELECT
	count(*) AS count,
	winner,
	counter * 60 * 5 AS counter
FROM
	(
		SELECT
			winner,
			round(length / 60 / 5)
				AS counter
		FROM
			players
		WHERE
			build = $1
			AND (
					hero = $2
					OR region = $3
				)
	)
GROUP BY
	winner, counter

42:
------------------------------------------
SELECT
	count(*) AS count,
	winner,
	counter * 60 * 5 AS counter
FROM
	(
		SELECT
			winner,
			round(length / 60 / 5)
				AS counter
		FROM
			players
		WHERE
			build = $1
			AND (hero = $2 OR region = $3)
	)
GROUP BY
	winner, counter

45:
---------------------------------------------
SELECT
	count(*) AS count,
	winner,
	counter * 60 * 5 AS counter
FROM
	(
		SELECT
			winner,
			round(length / 60 / 5) AS counter
		FROM
			players
		WHERE
			build = $1
			AND (hero = $2 OR region = $3)
	)
GROUP BY
	winner, counter

53:
-----------------------------------------------------
SELECT
	count(*) AS count,
	winner,
	counter * 60 * 5 AS counter
FROM
	(
		SELECT
			winner, round(length / 60 / 5) AS counter
		FROM
			players
		WHERE
			build = $1 AND (hero = $2 OR region = $3)
	)
GROUP BY
	winner, counter

58:
----------------------------------------------------------
SELECT
	count(*) AS count, winner, counter * 60 * 5 AS counter
FROM
	(
		SELECT
			winner, round(length / 60 / 5) AS counter
		FROM
			players
		WHERE
			build = $1 AND (hero = $2 OR region = $3)
	)
GROUP BY
	winner, counter

115:
-------------------------------------------------------------------------------------------------------------------
SELECT
	count(*) AS count, winner, counter * 60 * 5 AS counter
FROM
	(SELECT winner, round(length / 60 / 5) AS counter FROM players WHERE build = $1 AND (hero = $2 OR region = $3))
GROUP BY
	winner, counter

203:
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
SELECT count(*) AS count, winner, counter * 60 * 5 AS counter FROM (SELECT winner, round(length / 60 / 5) AS counter FROM players WHERE build = $1 AND (hero = $2 OR region = $3)) GROUP BY winner, counter


